ibm/mas_devops/roles/suite_app_install/tasks/visualinspection.yml

```diff
@@ -22,13 +22,12 @@
 # -----------------------------------------------------------------------------
 - name: Default Storage if not set by user
   when: mas_app_settings_visualinspection_storage_class is not defined or mas_app_settings_visualinspection_storage_class == ""
-  set_fact:
-    mas_app_settings_visualinspection_storage_class: "{{ lookup_storageclasses | ibm.mas_devops.defaultStorageClass(default_storage_classes_rwx_nogid) }}"
 
-- name: Assert that storage class has been defined
-  assert:
-    that: mas_app_settings_visualinspection_storage_class is defined and mas_app_settings_visualinspection_storage_class != ""
-    fail_msg: "mas_app_settings_visualinspection_storage_class must be defined"
+  vars:
+    # ROKS, OCS, AWS, Azure
+    supported_storage_classes: [ibmc-block-gold, ocs-storagecluster-ceph-rbd, gp2, efs, managed-premium]
```

Comment:
any reason to not use `ibm/mas_devops/common_vars/default_storage_classes.yml` to maintain the defaulted storage classes for MVI? It's probably easier to maintain this code in the future if we keep all the defaulted storage class in a centric place rather than keep it isolated in each playbook/role.

You could probably just create a new property specific to MVI there if that's the case.

Reply:
Worked with Caroline to use the default stoage class for MVI.